test/browser/specs/survey.md

```diff
@@ -0,0 +1,24 @@
+# Survey specification
```

Comment:
this is survey-create test. there will be more survey tests I imagine

Reply:
sure, I'll reorganize them